UI/window-basic-settings.cpp

```diff
@@ -1391,6 +1398,12 @@ void OBSBasicSettings::LoadSimpleOutputSettings()
 	int idx = ui->simpleOutRecFormat->findText(format);
 	ui->simpleOutRecFormat->setCurrentIndex(idx);
 
+	/* restrict list of bitrates  when multichannel is OFF*/
```

Comment:
Style:  Single line comments use `//`.  There is an extra space between "bitrates" and "when".